docs/admin/addons.md

```diff
@@ -0,0 +1,37 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
+You can install add-ons on a Kubernetes cluster using `kubectl apply -f`.
+
+This page lists some of the available add-ons.
+
+## Networking
+
+* [Weave Net](https://github.com/weaveworks/weave-kube) is a fast, reliable pod network that carries on working even in the face of network partitions, and doesn't have any infrastructure or database dependencies. Install it on a cluster whose kubelets are configured to expect a [CNI network plugin](/docs/admin/network-plugins/):
+
+      $ kubectl apply -f https://git.io/weave-kube
+
+  If you used [`kubeadm`](/docs/kubeadm/) to create your cluster, the kubelet will already be preconfigured to expect a CNI plugin, so you won't need to do anything special to get it Weave Net working.
+
+## Network Policy
+
+* [Weave Net Policy](https://github.com/weaveworks/weave-npc/tree/initial-implementation) extends Weave Net to support the [Kubernetes policy API](/docs/user-guide/networkpolicies/) so that you can securely isolate different pods from each other based on namespaces and labels:
+
+      $ kubectl apply -f https://raw.githubusercontent.com/weaveworks/weave-npc/40f5461f2f840eb8a223710e227f687cbaa55d0f/k8s/daemonset.yaml
```

Comment:
This isn't great. If you strongly want this, tag commit weaveworks/weave-npc@40f5461f2f840eb8a223710e227f687cbaa55d0f with a version like `v0.1.0` or `v1.0.1`